loan-payoff/loan-payoff-json-contract.md

```diff
@@ -93,17 +93,17 @@ Any and all errors should be conveyed via the following structure:
 ```
 ### Response Messages & Corresponding Logging Error Messages
 
-*See Logging Error Message section for additional details
+*All Errors beging with 'We're sorry we've encountered an error. Please contact your credit union.' followed by a Logging Error Message.
 
-| Client Error Message                                                                 | Logging Error Message                                                       | Notes |
-| ------------------------------------------------------------------------------------- |------------------------------------------------------------------------------------- |-----------------------------------------------|
-| We're sorry we've encountered an error. Please contact your credit union. | Duplicate loan type in CFG (BLT 1234) | This error represents issues with the configuration letter file setup |
-|| Duplicate loan type in CFG (VLT 1234) ||
-|| Duplicate loan type in CFG (SLT 1234) ||
-|| Error Opening Letterfile [config file name]: [system generated letter file read error message] ||
-| We're sorry we've encountered an error. Please contact your credit union. | Ineligible Loan Type | This type of error is for an ineligible loan |
-|| Payoff Days more than Maximum allowed ||
-|| Account Warning Found ||
-|| Loan Warning Found ||
-| We're sorry we've encountered an error. Please contact your credit union. | Loan Projection Error: [system generated PowerOn function LOANPROJECTCALC error message] | This type of error is a non-configuration file error |
-|| Fee Specfile Error: [Banno loan payoff fees PowerOn generated error message] ||
+| Logging Error Message                                                                | Additional Notes As Needed |
+|------------------------------------------------------------------------------------- |-----------------------------------------------|
+| Error Opening Letterfile [config file name]: [system generated letter file read error message] | This set of errors represents issues with the configuration letter file setup |
+| Duplicate loan type in CFG (BLT 1234) | In the CFG boat loan type has a duplicate loan type that is already in home, vehicle or secured loan type parameters. Remove the duplicates to resolve. |
+| Duplicate loan type in CFG (VLT 1234) | In the CFG vehicle loan type has a duplicate loan type that is already in home, boat or secured loan type parameters. Remove the duplicates to resolve.|
+| Duplicate loan type in CFG (SLT 1234) | In the CFG secured loan type has a duplicate loan type that is already in home, boat or vehicle loan type parameters. Remove the duplicates to resolve. |
+| Ineligible Loan Type | This set of errors is for ineligible loans |
+| Payoff Days more than Maximum allowed ||
+| Account Warning Found ||
+| Loan Warning Found ||
+| Loan Projection Error: [system generated PowerOn function LOANPROJECTCALC error message] | This set of errors is for non-configuration file errors |
```

Comment:
The "this set of errors...." statement seems a bit 'weird'. the other two 'headings' tells you what those errors represent whereas this one tells you what they don't represent. Technically, the 'this set of errors is for ineligible loans' header is also a 'non-configuration' file error. For consistency, maybe ..... "This set of errors is for skip payment processing errors" ??

Reply:
changed comment for loan payoff to
'This set of errors is for loan payoff processing errors'
and changed skip payment comment to
'This set of errors is for skip payment processing errors'